USAGE_desktop.md

```diff
@@ -646,3 +647,15 @@ gfxrecon.py capture -o vkcube.gfxr vkcube
 
 On Windows, after installing Python3, be sure to associate the `.py` file extension with
 the Python3 interpreter before you run the script.
+
+### Options Common To all Tools
+
+If the environment variable GFXRECON_NO_DEBUG_POPUP has any non-empty
```

Comment:
Recommend exact computing terms in prose have backticks, e.g. `` `GFXRECON_NO_DEBUG_POPUP` `` here and `` `abort()` `` below.

Reply:
Yes, good recommendation.  Fixed!